refresh.sh

```diff
@@ -1,19 +1,17 @@
 #!/bin/sh
 
-DNLOAD="./dnload.py"
-if [ ! -f "${DNLOAD}" ] ; then
-  echo "${0}: could not find dnload.py"
+DNLOAD="dnload"
+if [ ! -d "${DNLOAD}" ]; then
```

Comment:
As I said above, if the module is named dnload and one clones this project as dnload/, then, insofar as I can see, checking for the directory is not enough. We do not know where this project is cloned. Should probably check for dnload/__main__.py but I'll wait for your opinion.

Reply:
It seems double underscore is interpreted into bold. Ohwell.